app/views/profiles/edit.html.erb

```diff
@@ -52,6 +52,13 @@
         <button type="button" id="home_undelete" class="btn btn-outline-primary" hidden><%= t ".undelete" %></button>
       </div>
     </div>
+    <div class="my-2">
+      <%= f.text_field :location_name, :wrapper_class => "col-sm-4 d-flex flex-column pe-3", :class => "mt-auto", :id => "location_name" %>
+      <div id="location_name_warning" class="row align-items-center d-none">
+        <p class="m-0 w-auto fs-6 pe-1"><%= t ".location_name_warning" %></p>
+        <button id="location_default_name" class="btn btn-link p-0 w-auto text-sm-start" type="button"></button>
+      </div>
```

Comment:
This looks like a sentence and it's not obvious that it ends with a button I'm supposed to click to update the value:

![image](https://github.com/user-attachments/assets/7b059033-8584-457b-ba83-7b83e046c3b5)

Reply:
Was changed with the button named "Autofill %{country}".

Reply:
The language of %{country} is currently based on the browser settings / Accept-Language header. Is it what we want?

Reply:
Maybe we want something like this?
![image](https://github.com/user-attachments/assets/22f241ab-8afe-4031-9854-f0cf33c01b27)

"Autofill %{country}" is probably not going to look good because it's likely going to be composed of two different languages.

Reply:
and the button can become too large
![image](https://github.com/user-attachments/assets/17439310-4dc1-4a26-bf60-3143cb4cbaf2)

Reply:
Thanks for suggestions. I agree with you, it seems current solution has some UI problems, but adding additional disabled input doesn't align with the UI practices we are currently using. Maybe making it more minimalistic will be better:
We can remove button and have only following logic:
```
If user has empty `Location Name` field when entering `Profile Edit`:
    Until user changes `Location Name` field manually:
        Autofill will automatically fill `Location Name` field on pin change
Otherwise (if user has already saved `Location Name` and, therefore, field is filled, when entering `Profile Edit`):
    If `Location Name` field value exactly matches with the country name of the pin:
        Until user changes `Location Name` field manually:
            Autofill will automatically fill `Location Name` field on pin change
    Otherwise:
        Autofill will be deactivated
```

Reply:
> If `Location Name` field value exactly matches with the country name of the pin

That requires fetching the country from Nominatim before the location changes, possibly on page load, right?

Reply:
Yes, either on page load, or if we want to avoid an additional call on the page load, we can call Nominatim twice on the first change of the pin (one for the previous value and one for the new value, to compare them)